hero-openshift.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: page
+title: OpenShift in 30 mins
+permalink: /hero-openshift/
+---
+
+# Openshift in 30 mins with a Node.JS application  
+  
+## Prerequisites  
+
+These steps presume you already have Docker installed. If you don't, look at this quickstart docker guide, or install Docker from [here](insert-link). Before you proceed any further confirm you can run docker process commands by trying `docker ps`
+
+
+## Installing Openshift  
+  
+### Installing the oc cli tool  
+
+Download the Linux [oc binary](https://github.com/openshift/origin/releases) and place it in your path.
+
+
+## Bring your cluster up  
+
+Bring up your cluster for the first time by running:
+
+* `oc cluster up --public-hostname=127.0.0.1 --host-data-dir=$HOME/os/data-dir --host-config-dir=$HOME/os/config-dir`
+
+If successful you should see output like the following:
+
+![clusterup-terminal-output][image-clusterup]
+
+To bring up your cluster subsequently pass in the `--use-existing-config` flag to your `oc cluster up` command:
+
+* `oc cluster up --public-hostname=127.0.0.1 --host-data-dir=$HOME/os/data-dir --host-config-dir=$HOME/os/config-dir --use-existing-config`
+
+
+### Troubleshooting  
+
+If you encounter any errors the error messages are generally helpful enough to diagnose and resolve the issue. Some issues that were encountered included:
+
+* the docker service daemon/ service was not running or needed to be restarted
+
+
+### OpenShift console  
+
+Once you have successfully brought your cluster up you should be able to access the OpenShift console through the url specified at the end of your `oc cluster up` log. 
+
+![openshift-mainconsole][image-mainconsole]  
+
+![openshift-consoledashboard][image-dashboard]
+
+
+## Deploy your app to OpenShift  
+
+Deploy your app by executing:
+
+* `oc new-app <dockerhub-username>/<dockerhub-image-name>`
```

Comment:
It might be easier/better for new people if we just tell them to pick a docker image that we know works out of the box instead of assuming they have an image in the clou dalready.